### DISCUSSION TITLE
Help with using MIES

### DISCUSSION THREAD

**Discussion by jessewhite2007** [AllenInstitute/MIES#1108] [Q&A] [answered]
Hello, I'm new to coding and am going to be using MIES for electrophysiology experiments. Is there a manual or "how to" for using MIES for whole cell patch clamp? I can't figure out simple things like how to write my own protocols. Thanks

**Comment by timjarsky** [comment]
Hi @jessewhite2007 

Our [user documentation](https://alleninstitute.github.io/MIES/user.html) is limited but it does have some helpful content to get you started

I also made a short [video ](https://youtu.be/p_miGgyUMSg)demonstrating how to create a basic square pulse stimulus set. Once you understand how to add epochs and sweeps as shown in the video, it is fairly straightforward to make more complicated stimulus sets with the waveBuilder GUI.

**Comment by jessewhite2007** [comment]
Hi Dr. Jarsky,

I’m Catherine Kaczorowski and Kristen O’Connell’s RA that is doing patch seq. She gave me your email address soon after I posted that question. I was going to ask if you had a macros I could use. Catherine mentioned that you guys are using a new macros, I imagine it would be very similar to what I would need.

If not the video and documentation should be enough to get me started,

thanks!

Jesse

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: Tim ***@***.***>
Sent: Wednesday, October 13, 2021 9:21 PM
To: ***@***.***>
Cc: Jesse ***@***.***>; ***@***.***>
Subject: [EXTERNAL]Re: [AllenInstitute/MIES] Help with using MIES (#1104)


Hi @jessewhite2007<https://github.com/jessewhite2007>

Our user documentation<https://alleninstitute.github.io/MIES/user.html> is limited but it does have some helpful content to get you started

I also made a short video <https://youtu.be/p_miGgyUMSg> demonstrating how to create a basic square pulse stimulus set. Once you understand how to add epochs and sweeps as shown in the video, it is fairly straightforward to make more complicated stimulus sets with the waveBuilder GUI.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/AllenInstitute/MIES/issues/1104#issuecomment-942854111>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWBM42EKHYNFYCR6SSUUAMTUGYWBXANCNFSM5F5S3LTA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

---

The information in this email, including attachments, may be confidential and is intended solely for the addressee(s). If you believe you received this email by mistake, please notify the sender by return email as soon as possible.

**Comment by timjarsky** [comment]
@jessewhite2007 it is possible to get a relatively recent version of the IVSCC production stimulus sets from within MIES; 
![image](https://user-images.githubusercontent.com/5693697/137235909-0032367f-f88a-4048-9da7-203001a3323e.png)

**Comment by jessewhite2007** [comment, marked as answer]
Thanks, I’ll try it out today

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: Tim ***@***.***>
Sent: Wednesday, October 13, 2021 9:38 PM
To: ***@***.***>
Cc: Jesse ***@***.***>; ***@***.***>
Subject: [EXTERNAL]Re: [AllenInstitute/MIES] Help with using MIES (#1104)


@jessewhite2007<https://github.com/jessewhite2007> it is possible to get a relatively recent version of the IVSCC production stimulus sets from within MIES;
[image]<https://user-images.githubusercontent.com/5693697/137235909-0032367f-f88a-4048-9da7-203001a3323e.png>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/AllenInstitute/MIES/issues/1104#issuecomment-942859918>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWBM42G4CHRUP5JFN6UT533UGYYBJANCNFSM5F5S3LTA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

---

The information in this email, including attachments, may be confidential and is intended solely for the addressee(s). If you believe you received this email by mistake, please notify the sender by return email as soon as possible.
